packages/client/README.md

```diff
@@ -130,22 +145,7 @@ sequenceDiagram
 In (1) the Client sends a request to the provider backend in order to get all values from all feature flags that it has.
 Once the provider backend replies (2) the client holds all flag values and therefore the flag evaluation process is synchronous.
 
-In order to prevent flag evaluation to the default value while flags are still being fetched, it is highly recommended to only look for flag value after the provider has emitted the `Ready` event.
-The following code snippet provides an example.
-
-```ts
-import { OpenFeature, ProviderEvents } from '@openfeature/web-sdk';
-
-OpenFeature.setProvider( /*set a provider*/ );
-
-// OpenFeature API
-OpenFeature.addHandler(ProviderEvents.Ready, () => {
-  const client = OpenFeature.getClient();
-  const stringFlag = client.getStringValue('string-flag', "default value"))
-
-  //use stringFlag from this point
-});
-```
+In order to prevent flag evaluations from defaulting while the provider is initializing, it is highly recommended to only look for flag value after the provider is ready. This can be done using the `setProviderAndWait` method or using the `setProvider` method and listening for the `READY` [event](#eventing).
```

Comment:
```suggestion
In order to prevent flag evaluations from defaulting while the provider is initializing, it is highly recommended to evaluate flags only after the provider is ready. This can be done using the `setProviderAndWait` method or using the `setProvider` method and listening for the `READY` [event](#eventing).
```